### ISSUE TITLE
Test do not work, the last token is always missing

### ISSUE THREAD

**Issue by padmalcom** [AristotelisPap/Question-Answering-with-BERT-and-Knowledge-Distillation#1]
Hi, first of all: Thanks for your code. I tried to reproduce your results but in the answer, there is always a missing word. The example:

```
text = "My house is in Los Angeles."
question = "Where do I live ?"
```

returns _in los_, not _in los angeles_.

When I change:

`answer_tokens = all_tokens[torch.argmax(logits.start_logits[0][st:]) :torch.argmax(logits.end_logits[0][st:])+1]`

to:

`answer_tokens = all_tokens[torch.argmax(logits.start_logits[0][st:]) :torch.argmax(logits.end_logits[0][st:])+2]`

The answers seem to be correct but I cannot tell why the +1 is wrong here.
